lib/locales/es.json

```diff
@@ -0,0 +1,434 @@
+        "searchForFriends": "Buscar amigos",
+        "inviteToChat": "Invitar al chat",
+        "inviteToWhiteNoise": "Invitar a White Noise",
+        "groupChatDetails": "Detalles de conversación de grupo",
+        "chatDetails": "Detalles de conversación",
+        "connectAnotherProfile": "Conectar otro perfil",
+        "loginWithExistingProfile": "Iniciar sesión con perfil existente",
+        "createNewProfile": "Crear nuevo perfil",
+        "profiles": "Perfiles",
+        "contactsNotReadyForSecureMessaging": "Estos contactos aún no están listos para mensajería segura. ¡Comparte White Noise con ellos para comenzar!",
+        "userProfile": "Perfil de usuario",
+        "support": "Soporte",
+        "errorLoadingUserKeyPackage": "Error al cargar el paquete de llaves del usuario",
+        "chatStartedSuccessfully": "Chat con {name} iniciado exitosamente",
+        "failedToCreateDirectMessageGroup": "Error al crear el grupo de mensajes directos",
+        "unfollowed": "Dejaste de seguir a {name}",
+        "followed": "Siguiendo a {name}",
+        "noUserToAddToGroup": "No hay usuario para agregar al grupo",
+        "unfollow": "Dejar de seguir",
+        "follow": "Seguir",
+        "creatingChat": "Creando chat...",
+        "startChat": "Iniciar chat",
+        "addRelay": "Agregar relé",
+        "addNewRelay": "Agregar nuevo relé",
+        "removeRelay": "Eliminar relé",
+        "addToGroup": "Agregar al grupo",
+        "addToAnotherGroup": "Agregar a otro grupo",
+        "removeFromGroup": "Eliminar del grupo",
+        "sendMessage": "Enviar mensaje",
+        "searchChat": "Buscar en chat",
+        "editGroup": "Editar grupo",
+        "members": "Miembros",
+        "member": "Miembro",
+        "admin": "Administrador",
+        "groupRelays": "Relés del grupo",
+        "noRelaysConnected": "No hay relés conectados",
+        "noAccountsFound": "No se encontraron cuentas",
+        "errorLoadingProfile": "Error al cargar el perfil",
+        "wallet": "Billetera",
+        "groupNotFound": "Grupo no encontrado",
+        "invitationNotFound": "Invitación no encontrada",
+        "errorLoadingRelays": "Error al cargar relés",
+        "connectWallet": "Conectar billetera",
+        "viewQrCode": "Ver código QR",
+        "whatWalletCanIConnect": "¿Qué billetera puedo conectar?",
+        "publishNewKeyPackage": "Publicar nuevo paquete de llaves",
+        "inspectRelayKeyPackages": "Inspeccionar paquetes de llaves del relé",
+        "deleteAllKeyPackagesFromRelays": "Eliminar todos los paquetes de llaves de los relés",
+        "switchAccount": "Cambiar cuenta",
+        "scanQrCode": "Escanear código QR",
+        "removeContact": "Eliminar contacto",
+        "signedOutChooseDifferent": "Sesión cerrada. Elige un perfil diferente.",
+        "failedToLoadAccounts": "Error al cargar las cuentas"
+    },
+    "errors": {
+        "failedToChangeLanguage": "No se pudo cambiar el idioma a",
```

Comment:
`"No se pudo cambiar el idioma a {languageName}"`
I seems to me that here a param is needed, in some languages the language name could be in the middle of the sentence I guess. It's not the case of spanish, here the language name at the end it's ok. I noticed this because at first I though that the last `a` was typo, then realized that it was a sentence to be continued...